src/app/views/sessions/signin/signin.component.ts

```diff
@@ -382,6 +384,16 @@ export class SigninComponent implements OnInit, OnDestroy, AfterViewInit {
 
   successLogin(): void {
     this.snackBar.dismiss();
+
+    if (!environment.production) {
```

Comment:
Best to find a way to enable this for nightly builds only.
If too hard, just switch the statement here.